server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMessagesCreationProcessor.java

```
@@ -173,6 +176,8 @@ private void handleCreate(CreationMessageEntry create, Builder responseBuilder,
                         .type("error")
                         .description("unexpected error")
                         .build());
+        } catch (IllegalStateException e) {
+
```

an empty catch block at least requires a comment to explain why